packages/features/bookings/components/PeopleFilter.tsx

```diff
@@ -11,14 +11,22 @@ import { useLocale } from "@calcom/lib/hooks/useLocale";
 import { trpc } from "@calcom/trpc/react";
 import { AnimatedPopover, Avatar, Divider, FilterSearchField, Icon, Button } from "@calcom/ui";
 
+import useMeQuery from "../../../trpc/react/hooks/useMeQuery";
+
 export const PeopleFilter = () => {
   const { t } = useLocale();
-
+  const { data: me } = useMeQuery();
```

Comment:
Can we useSession here to get current userId instead of hitting the useMeQuery which is a redudant api call as we have this data in session?